modules/ROOT/pages/to-create-an-asset.adoc

```diff
@@ -197,8 +196,23 @@ To create a ruleset asset:
 . In Exchange, select *Publish new asset*.
 . Enter a name for the asset.
 . From the drop-down list, select the asset type *Ruleset*.
-. If you want to select a file to upload, click *Choose file*.
-. If you want to edit the advanced settings, *GroupId*, *AssetId*, and *Version* (GAV), click *Advanced*. 
+. To upload, click *Choose file*.
+. To edit the advanced settings, *GroupId*, *AssetId*, and *Version* (GAV), click *Advanced*. 
++
+Exchange generates the group ID, asset ID, and version for you, and you can change these values as needed. Use the advanced settings to change the asset version.
+. Select an option for *Lifecycle state*. The default is *Stable*.
+. Select *Publish*.
+
+== Create an AsyncAPI Asset
+An AsyncAPI is an AsyncAPI specification file that specifies an event-driven API. You can create AsyncAPIs with Avro schemas. The file types for AsyncAPIs with Avro schemas are JSON (.json) and AVSC (.avsc).
```

Comment:
File types for AsyncAPIs are only `.json` and `.yaml`. `.avsc` is only for AVRO fragments